src/App.tsx

```diff
@@ -2,7 +2,8 @@ import React, { useState } from 'react'
 import { Container, Box, Typography, TextField, Button } from '@mui/material'
 import { Stack } from '@mui/system'
 import { styled } from '@mui/material/styles'
-
+let timer: any
```

Comment:
Instead use `any`, you can use `ReturnType<typeof setInterval>`, it may not a best answer but at least better than `any`.

I googled keywords `typescript type of interval`, the second record: https://www.designcise.com/web/tutorial/what-is-the-correct-typescript-return-type-for-javascripts-setinterval-function